### ISSUE TITLE
learning rate 

### ISSUE THREAD

**Issue by tengolagan** [SenHe/Flow-Style-VTON#41]
How come ResUnetGenerator doesn't have a method update_learning_rate? 

Last 3 rows of code in train_PBAFN_e2e_fs.py : 
 
<img width="522" alt="Capture d’écran 2023-04-13 à 22 38 08" src="https://user-images.githubusercontent.com/67476919/231877466-fa4086c8-dc54-403e-8c27-d59a9e6df2e8.png">

Why are we updating model (the warping model) twice ? Shouldn't **model.module.update_learning_rate(optimizer_gen)** be something like **model_gen.module.update_learning_rate(optimizer_gen) ?**
